### DISCUSSION TITLE
Symbols special characters mapping

### DISCUSSION THREAD

**Discussion by KiraUnderwood** [jsvine/pdfplumber#886] [Q&A]
Hi All!
I have a small issue with the library.
My PDF contains special characters, which in Word are identified as 'Symbols' Font. This font is also detected by pdfplumber for those characters.
Even though I see that Adobe provides spec of mapping this font to Unicode, pdfplumber still uses standard encoding and replaces 'more than or equal' and 'less than of equal' to pound and double dagger signs.
Can I as a user of this library do something to preserve my symbols? 
I found only one library, tabula-py which preserves them, but it is Java originally and also it reads everything into tables which I may not need.
Thank you,
Kira

**Comment by jsvine** [comment]
Hi @KiraUnderwood, and thanks for your interest in this library. Are you able to share the PDF?

**Comment by KiraUnderwood** [comment]
Hi! Yes, sure.
I've created it myself via word and using Symbols .
The library read it as SymbolsMT though...
I dont know how to share the file here.
But you may just paste this to Word and save it as PDF:
10  x  20
20  y 40

**Reply by KiraUnderwood** [reply]
oops, it does not work on pasting too...

**Reply by KiraUnderwood** [reply]
I've uploaded it to one of my repos:
https://github.com/KiraUnderwood/web/blob/master/example_pdf.pdf

**Reply by jsvine** [reply]
Thanks for sharing that, @KiraUnderwood. That's helpful. I don't think you'll be able to get `pdfplumber` to render those characters _in the way you expect them to_ in plain text. Generally speaking, the information available to `pdfplumber` is the unicode representation _in the PDF_, rather than their visual "glyph" representation. (I would assume this is the case for most other tools, and I'm surprised that it worked in Tabula.)

**Reply by KiraUnderwood** [reply]
@jsvine Thanks a lot for the reply! That's what I suspected actually, since I tried other python libs too...thank you for confirming.
